.github/workflows/unit-test.yml

```
@@ -1,9 +1,9 @@
 # This workflow will install Python dependencies, run tests and lint with a single version of Python
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: Continuous Integration
+name: CI - Unit Tests
```

```suggestion
name: CI - Unit Tests & Linting
```
On lint également avec flake8, on ne fait pas uniquement tourner des tests, donc je suppose qu'on devrait aussi le spécifier dans le titre de l'action.